db/seeds.rb

```diff
@@ -1,11 +1,15 @@
 require 'nokogiri'
 require 'open-uri'
 
+user = User.create(email: 'test@gmail.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.